table/queries/merged_reduced_scans.sql

```diff
@@ -57,6 +61,7 @@ CREATE TEMP FUNCTION ClassifyError(error STRING, source STRING) AS (
     WHEN ENDS_WITH(error, "readLoopPeekFailLocked: <nil>") THEN "tls/tls.failed"
     WHEN ENDS_WITH(error, "missing ServerKeyExchange message") THEN "tls/tls.failed"
     WHEN ENDS_WITH(error, "no mutual cipher suite") THEN "tls/tls.failed"
+    WHEN REGEXP_CONTAINS(error, "TLS handshake timeout") THEN "tls/timeout"
```

Comment:
Why is this s regex now?
Isn't there a simple string contains?
Same for the other ones

Reply:
Bigquery only has REGEXP_CONTAINS and REGEXP_MATCH.

Reply:
ENDS_WITH/STARTS_WITH are used when the string only appears at the end/beginning of an error. If it appears in the middle we have to use REGEXP_CONTAINS.

Reply:
There's also STRPOS. It may be worth comparing the performance, since BigQuery is a huge cost, but I'm fine either way.

Reply:
I tried timing it and the strpos version took 3% longer. So probably the query planner is doing something smart and they're basically equivalent.